service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToSubmitTest.java

```diff
@@ -60,8 +60,8 @@ void mocking() {
 
         given(userDetailsService.getUserName())
             .willReturn("Emma Taylor");
-        given(documentGeneratorService.generateSubmittedFormPDF(any(), any()))
-            .willReturn(pdf);
+        given(caseSubmissionService.generateSubmittedFormPDF(any()))
```

Comment:
we should use an argument captor and assert that data is we are sending to docmosis is correct

Reply:
Do we care about the data here, or just that something has been generated and that we can trust the unit tests that the data will have been produced correctly?